testdata/project-v3-config/Dockerfile

```diff
@@ -15,7 +19,11 @@ COPY api/ api/
 COPY controllers/ controllers/
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+# the GOARCH has not a default value to allow the binary be built according to the host where the command
```

Comment:
Same link to consider: https://go.dev/doc/install/source#environment

Reply:
It was added in the docs.
I do not want to encourage users to change that in the dockerfile for the common cases.

Reply:
Gotcha, sounds good!